src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseAmendmentExtensions.cs

```diff
@@ -27,9 +27,18 @@ public static Release CreateAmendment(this Release release, DateTime createdDate
 
             var context = new Release.CloneContext(amendment);
 
+            // Copy ReleaseContentSections/ContentSections and ContentBlocks associated with those sections
             amendment.Content = amendment
                 .Content
-                .Select(rcs => rcs.Clone(context))
+                .Select(rcs => rcs.Clone(context)) // Old/New ContentBlock pairs added to context here
```

Comment:
One of these comments saying `ContentBlock pairs` I'm guessing needs a slight tweak.

Reply:
Answered in other comment.